exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,17 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
```

Comment:
This is a very safe method of solving this, I avoided using conditionals because it will first check if it greater or equal to 80 and since it's already checked that you can make your second if statement just check if it is 60(It's a logical operator in itself). I hope my explanation makes sense.